tests/modeladmin/tests.py

```diff
@@ -812,6 +812,50 @@ class BandAdmin(ModelAdmin):
             ["extra", "transport", "id", "DELETE", "main_band"],
         )
 
+    def test_multiple_log_actions(self):
+        ma = ModelAdmin(Band, self.site)
+        mock_request = MockRequest()
+        mock_request.user = User.objects.create(username="akash")
+        content_type = get_content_type_for_model(self.band)
+
+        flag = DELETION
+        Band.objects.create(
+            name="The Beatles",
+            bio="A legendary rock band from Liverpool.",
+            sign_date=date(1962, 1, 1),
+        )
+        Band.objects.create(
+            name="Mohiner Ghoraguli",
+            bio="A progressive rock band from Calcutta.",
+            sign_date=date(1967, 1, 1),
+        )
+        queryset = Band.objects.all().order_by("-id")[:2]
+        with self.assertNumQueries(2):
```

Comment:
ah that's right it's 2 - one for the bulk create + one for the content type retrieval.

I'm now thinking that to make it clear that it's not n+1ing that we add another Book. It's probably also worth adding a note to remind maintainers that the 2 is for the create + content type.